__mocks__/fetch.ts

```diff
@@ -108,6 +108,26 @@ const requestMock: any = vi.fn().mockImplementation((uri, params) => {
         delete sessionResponse.capabilities.browserName
     }
 
+    if (
+        body &&
+        body.capabilities &&
+        body.capabilities.alwaysMatch.mobileMode &&
+        body.capabilities.alwaysMatch.windowsAppMode
```

Comment:
```suggestion
body?.capabilities?.alwaysMatch.mobileMode &&
body?.capabilities?.alwaysMatch.windowsAppMode
```

Reply:
Same goes for the code below

Reply:
I can change it,m but then i need to change it for the complete file 😅 , I used the same style